src/alloc.c

```diff
@@ -175,49 +200,47 @@ JL_DLLEXPORT int jl_field_index(jl_datatype_t *t, jl_sym_t *fld, int err)
 
 JL_DLLEXPORT jl_value_t *jl_get_nth_field(jl_value_t *v, size_t i)
 {
-    jl_datatype_t *st = (jl_datatype_t*)jl_typeof(v);
+    jl_datatype_t *st = (jl_datatype_t *)jl_typeof(v);
     assert(i < jl_datatype_nfields(st));
-    size_t offs = jl_field_offset(st,i);
-    if (jl_field_isptr(st,i)) {
-        return *(jl_value_t**)((char*)v + offs);
+    size_t offs = jl_field_offset(st, i);
+    if (jl_field_isptr(st, i)) {
+        return *(jl_value_t **)((char *)v + offs);
     }
-    return jl_new_bits(jl_field_type(st,i), (char*)v + offs);
+    return jl_new_bits(jl_field_type(st, i), (char *)v + offs);
 }
 
 JL_DLLEXPORT jl_value_t *jl_get_nth_field_checked(jl_value_t *v, size_t i)
 {
-    jl_datatype_t *st = (jl_datatype_t*)jl_typeof(v);
-    if (i >= jl_datatype_nfields(st))
-        jl_bounds_error_int(v, i+1);
-    size_t offs = jl_field_offset(st,i);
-    if (jl_field_isptr(st,i)) {
-        jl_value_t *fval = *(jl_value_t**)((char*)v + offs);
-        if (fval == NULL)
-            jl_throw(jl_undefref_exception);
+    jl_datatype_t *st = (jl_datatype_t *)jl_typeof(v);
+    if (i >= jl_datatype_nfields(st)) jl_bounds_error_int(v, i + 1);
```

Comment:
Yes. In fact, I explicitly enabled single-line ifs (and case labels and loops) since this seemed the default in Julia. I'll disable these again.

Actually -- what about case labels? Should they remain allowed, as in
```C
case 42: return "42";
```